src/languageHandler.tsx

```diff
@@ -88,8 +88,25 @@ export function _td(s: string): string { // eslint-disable-line @typescript-esli
  * */
 const translateWithFallback = (text: string, options?: object): { translated?: string, isFallback?: boolean } => {
     const translated = counterpart.translate(text, { ...options, fallbackLocale: counterpart.getLocale() });
-    if (!translated || /^missing translation:/.test(translated)) {
+    if (!translated || translated.startsWith("missing translation:")) {
         const fallbackTranslated = counterpart.translate(text, { ...options, locale: FALLBACK_LOCALE });
+        if ((!fallbackTranslated || fallbackTranslated.startsWith("missing translation:"))
```

Comment:
Just to triple check intent, these brackets that were added actually changed the behaviour of the `if` - `&&` has higher precedence than `||`

Reply:
noted, the PR has been a litany of careless errors unfortunately. 

the intent is to never enter the if clause if we are in development mode (i.e. leave the behaviour as-is)